Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitSendBinding.cs

```diff
@@ -101,17 +102,10 @@ public async Task Send(string modelCardId)
         throw new SpeckleSendFilterException("No objects were found to convert. Please update your publish filter!");
       }
 
-      var sendInfo = new SendInfo(
-        modelCard.AccountId.NotNull(),
-        modelCard.ProjectId.NotNull(),
-        modelCard.ModelId.NotNull(),
-        _revitSettings.HostSlug.NotNull()
-      );
-
       var sendResult = await sendOperation.Service
         .Execute(
           revitObjects,
-          sendInfo,
+          modelCard.GetSendInfo(_revitSettings.HostSlug.NotNull()),
```

Comment:
the pattern does seem to be there for revit?

Reply:
partly yes, but still this HostSlug in settings depends on old kit classes which we will change it.